### DISCUSSION TITLE
Dose kotlindl support Kotlin Native?

### DISCUSSION THREAD

**Discussion by jihaohaaaa** [Kotlin/kotlindl#403] [Ideas]
Dose kotlindl support Kotlin Native? Or will it support Kotlin Native in the future? (Maybe it could be support by using Tensorflow c++ ABI to imporve the speed)

**Comment by professorDeveloper** [comment]
> Dose kotlindl support Kotlin Native? Or will it support Kotlin Native in the future? (Maybe it could be support by using Tensorflow c++ ABI to imporve the speed)



> Dose kotlindl support Kotlin Native? Or will it support Kotlin Native in the future? (Maybe it could be support by using Tensorflow c++ ABI to imporve the speed)

Yea  Kotlin Support Native   kotlin android developer  native programming.

**Comment by zaleslaw** [comment]
For now, we do nothing with Kotlin Native and have no such plans
The main problems with Tensorflow C++ API for us are some missing parts related to the gradients
